tox.ini

```diff
@@ -40,7 +40,7 @@ passenv =
     XDG_CONFIG_HOME
     XDG_RUNTIME_DIR
 commands =
-    pytest -vv tests/test_{envname}.py --junitxml={toxinidir}/junit_{envname}.xml []
+    pytest -vv tests/test_{envname}.py --junitxml={toxinidir}/junit_{envname}.xml --pytest-container-log-level=debug []
```

Comment:
is this a debugging leftover? I think in general that is too verbose for my personal liking.